steward/src/cellars/adaptors/vesting_simple.rs

```diff
@@ -1,14 +1,14 @@
 use ethers::{abi::AbiEncode, types::Bytes};
-use steward_abi::vesting_simple_adaptor::VestingSimpleAdaptorCalls;
-use steward_proto::steward::vesting_simple_adaptor;
+use steward_abi::vesting_simple_adaptor_v2::VestingSimpleAdaptorV2Calls as AbiVestingSimpleAdaptorV2Calls;
+use steward_proto::steward::vesting_simple_adaptor_v2;
 
 use crate::{
     error::Error,
     utils::{sp_call_error, sp_call_parse_address, string_to_u256},
 };
 
-pub(crate) fn vesting_simple_adaptor_v1_call(
-    params: steward_proto::steward::VestingSimpleAdaptorCalls,
+pub(crate) fn vesting_simple_adaptor_v2_calls(
```

Comment:
We're renaming v1 to v2 here -- is the implication that nothing will ever need to call v1 again?

Reply:
yes